packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/http
 
+## 1.2.0
+
+### Minor Changes
+
+- The return type for `createSubOrganization` has been updated to include a `privateKeys` field to reflect that it now calls createSubOrganizationV3
```

Comment:
@r-n-o is this correct?

Reply:
Kind of! I think we should be more concrete/explicit about what's changing:
* calling `.createSubOrganization` on our HTTP client will trigger an activity of type `CREATE_SUB_ORGANIZATION_V3` instead of `CREATE_SUB_ORGANIZATION_V2` previously
* If there are any policies referencing `CREATE_SUB_ORGANIZATION_V2` specifically, they won't work out of the box. These policies will need to be updated to allow `CREATE_SUB_ORGANIZATION_V3`
* `CREATE_SUB_ORGANIZATION_V3` supports everything `CREATE_SUB_ORGANIZATION_V2` supports, with the addition of a `privateKeys` field to atomically create a sub-org and private keys
* When reading `createSubOrganization` results, SDK users need to look at `activity.result.createSubOrganizationResultV3` instead of `activity.result.createSubOrganizationResult` previously

Reply:
will update the readme in my PR